j2v/str_templates/looker_templates.py

```diff
@@ -3,6 +3,7 @@
     description: \"{__desc}\"
     type: {looker_type}
     sql: ${{TABLE}}.{__path}::{json_type} ;;
+    {group_label_string}
```

Comment:
Can we avoid the empty line in case the group label is an empty string?

Reply:
Sure I'll fix this in the next commit